### PR TITLE
Complete overhaul for build images

### PR DESCRIPTION
**Pull request by shokakucarrier**
Switched to UBI9 and completely ditched JDK 1.8 and Python2
use latest instead of jdk11 tag for base image because JDK 1.8 version no longer exist
removed setting users process for indy
enhancement of readability for indy Dockerfile
removed dumb-init
removed unnecessary packages in base image
removed redundant files

**Require rebuild for every services**

### CODE REVIEW

**Comment by ligangty** [comment]
@shokakucarrier I've created a "ubi9" branch from master, can we use this branch to test these image changes?

**Comment by shokakucarrier** [comment]
> @shokakucarrier I've created a "ubi9" branch from master, can we use this branch to test these image changes?

Done

File: indy/Dockerfile [hunk 1]
```diff
@@ -59,11 +46,16 @@ RUN chgrp -R 0 /opt && \
     chmod -R g=u /usr/share/indy && \
     chgrp -R 0 /home/indy && \
     chmod -R g=u /home/indy && \
-    chown -R 1001:0 /home/indy && \
-    chmod 644 /etc/profile.d/setup-user.sh
+    chown -R 1001:0 /home/indy
+
+EXPOSE 8080 8081 8000
 
 USER 1001
```
Comment (ligangty) [inline indy/Dockerfile:53]: So just curious that where is user 1001 is created and mapped?

Comment (shokakucarrier) [reply indy/Dockerfile:53]: it doesn't matter, this is just for OpenShift arbitary user execution, I tested this on GPC it works with this line and without user wrapper script

**Review by ligangty — APPROVED** [review]
Generally lGTM. Let's use this for devel testing after merge.

**Pull request merged**